### ISSUE TITLE
Ability configure proxy settings to be set in Postman app for the connection to Postman Servers?

### ISSUE THREAD

**Issue by asi-postman** [a85/POSTMan-Chrome-Extension#7705]
**Is your feature request related to a problem? Please describe.**
The ability to configure proxy settings to be set in Postman app for the connection to Postman Servers?

**Describe the solution you'd like**
Access within the Postman App to connect to Postman servers

**Comment by ioneyed** [comment]
This is primarily around the fact that the current "settings > proxy" are not taken into account for the "Sign-in" or "Sync" modules within the application. Unfortunately I am behind an authenticated proxy for all traffic which limits that interactions. I have tried to leverage windows proxy, mac proxy, linux proxy settings and without having full root access (corporate environment) and setting up a fully transparent local proxy it is hindering usage of Postman.

We have been able to setup a local proxy that chains to the corporate proxy with the authentication which allows windows proxy, mac's proxy, and linux proxy to work as a chain without being "transparent" however it is only feasible for more technical workers to setup a local forward proxy. We are rolling out Postman to all walks of work including Product Managers, Upper Management, and other non-knowledge workers for various tasks, monitoring, and workflow automation.
